Library/Homebrew/formula.rb

```diff
@@ -1599,19 +1599,22 @@ def mirror(val)
     # handled for you by the Brew Test Bot.
     #
     # <pre>bottle do
-    #   root_url "http://mikemcquaid.com" # Optional root to calculate bottle URLs
+    #   root_url "https://example.com" # Optional root to calculate bottle URLs
```

Comment:
You were replaced in the other bottle example we keep lying around the other week. Just mimicking that change, promise :wink: